src/shared/locales/enUs/translations/patient/index.ts

```diff
@@ -146,7 +146,7 @@ export default {
       label: 'Visits',
       startDateTime: 'Start Date',
       endDateTime: 'End Date',
-      type: 'Type',
+      type: 'Visit Type',
```

Comment:
Just curious why we decide to add Visit here? Aren't we already looking at Visit* fields? if we have to do it for one should we maybe not also do it for all of them? i.e. Visit Start Date, Visit End Date etc for consistency? Also not sure if this is related to the issue that this PR is meant to be addressing? I stand corrected should it be.

Reply:
What I was originally thinking is to create a translation key for the visit type so that it can be used in the new request imaging page 
 `<div className="visits">
         {newImagingRequest.patient && visitOption ? (
                <SelectWithLabelFormGroup
                  name="visit"
                  label={t('patient.visits.type')}` 
![image](https://user-images.githubusercontent.com/40738727/90403384-e9195f80-e06e-11ea-8758-4f43706a7cd9.png)


Maybe it will make more sense to create a new translation key in the imaging locale index.ts file to prevent some confusion

Reply:
You can keep the translation key in patients since visits are part of the patients module. Maybe just call it _Visit_ rather than _Visit Type_ because visit type makes me think (outpatient/inpatient/emergency etc) where here we are giving the user a list of all visits for this patient regardless of the type of visit it is, what do you think?

Reply:
On it